docs/TRANSLATING.md

```diff
@@ -2,13 +2,29 @@
 
 This guide is written to provide instructions from scratch for contributing to the translation of the Jamulus application to other languages.
 
-The code for Jamulus is open source, and is managed and made available via the Github site.
+The code for Jamulus is open source, and is managed and made available via the Github site. However, translators are not required to use GitHub. Jamulus is available on [Hosted Weblate](https://hosted.weblate.org/projects/jamulus/).
```

Comment:
```suggestion
The Jamulus source code is libre software, managed and made available via GitHub. However, translators are not required to use GitHub. The Jamulus translation effort is conducted on [Hosted Weblate](https://hosted.weblate.org/projects/jamulus/).
```

Reply:
I think we agreed that we call Jamulus Free and open source somewhere, https://github.com/jamulussoftware/jamuluswebsite/issues/42 hints at it.

Reply:
It is short of what is a meaningful distinction from only free software/open source.
Also, errors in translation occurs with "free", and "open source" isn't a trademark in other languages.
Thereby the issue of not focusing on freedoms sells it short of even free software.
Free software in turn gets mistaken for freeware.

Reply:
See https://jamulus.io:

> Since Jamulus is [free and open source software](https://www.gnu.org/philosophy/free-sw.en.html) (FOSS) licensed under the [GPL](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html), you can help us!

was the result on what we decided. So it should be comparable here (not copylefted libre software which is technically true, but is yet another phrase)

Reply:
It doesn't work anywhere, so whether it is temporarily consistent is a secondary concern.
https://remmina.org/ works
https://www.jamovi.org/ and https://jamulus.io/ does not work

People don't know what the GPL is, and even if they did, there are three versions of that license.
There is also "Play music online. With friends. For free." which makes a bigger point out of it being gratis, which undersells freedom.
The landing side is not a place where the decisions made command any sort of authority.
Mac should be "macOS". (like it is in the side-menu)
Erroneous use of an ampersand (&).
Triple self-referencing (and then som), with redundant info in the ingress, not line-shifted at that.
 and brings → bringing
Wall of text inducing stress.
Exclamation points after two consecutive sentences for no good reason.
Duplicated button making the website more likely to scroll vertically.
"ASIO is a trademark and software of Steinberg Media Technologies GmbH " should be in a mouse-over.

How about reducing the workload to just the work being done and then look at it all once done to see if something can be improved upon?

Reply:
Thanks for raising these issues. I've opened an issue on the documentation repo. 

> How about reducing the workload to just the work being done and then look at it all once done to see if something can be improved upon?

As it's part of the main repo and mainly for contributors, I think we could also remove it fully as it's probably not even necessary. The GPL is part of the repo, linked on Weblate and FLOSS is pretty obvious.

Reply:
Suggestion: 

Jamulus managed and made available via GitHub. However, translators are not required to use GitHub...

Reply:
"is"?
"Translators need not use GitHub for anything beside ___"

Reply:
Ok. Even shortened it a bit more.

Applied.

Reply:
> "Translation thereof can be done without the use of GitHub, whereas fixing source strings yourself(?) requires it."

Yes (although the installer should support it). I think the source strings issue should be fixed, but I'm not sure if and how Qt supports it.